scripts/docker-testnet/build.sh

```diff
@@ -0,0 +1,6 @@
+pushd ../..
```

Comment:
add `#!/usr/bin/env bash` here also? i think pushd is bash specific

Reply:
is this script intended to be run separately?

Reply:
yes, it is. I could have added the build step inside the `./start.sh` script, but it was easier for me to debug, since it didn't build the images at every run. Don't have a strong opinion on this, I can add it to the wrapper script instead. I was planning on adding a `README.md` nevertheless.

Reply:
I think it is better that we keep the build separate from the run script. Not a strong opinion, thus.